raiden/messages.py

```diff
@@ -1832,6 +1832,89 @@ def verify_request_monitoring(
         )
 
 
+class SettlementRequest(Message):
```

Comment:
Lets rename this to `SettlementLightRequest`, in order to know that this class is related with light clients

Reply:
`/light_channels` seems better as a path